cpp/leaky_libc.cc

```diff
@@ -86,19 +86,17 @@ List<Str*>* regex_match(Str* pattern, Str* str) {
   int outlen = pat.re_nsub + 1;  // number of captures
 
   const char* s0 = str->data_;
-  regmatch_t* pmatch =
-      static_cast<regmatch_t*>(malloc(sizeof(regmatch_t) * outlen));
-  int match = regexec(&pat, s0, outlen, pmatch, 0) == 0;
+  Slab<regmatch_t>* pmatch = NewSlab<regmatch_t>(outlen);
```

Comment:
Ah I think this one can be left alone because there's no leak?  Sorry I misphrased the goal -- get rid of leaks, not necessarily malloc

I think malloc and free() in the same function happens to be OK

Though interested in any feedback.

One way we could verify this is to make sure there is test coverage, and that ASAN catches no leaks when running the test (which we need to do for all tests)

Reply:
Ah great. I was wondering if this on was necessary. I'll drop it